### ISSUE TITLE
code cleanup

### ISSUE THREAD

**Issue by notEvil** [hut/ranger#251]
I am quite fond of python and therefore don't like code that is obviously influenced by other languages. As an example: container/directory.py - class Directory
There you 'declare' all variables at the very beginning as if it was a java/C++ class. But in fact you create class variables (~= static). Those are of course accessible using self.something, but this is considered bad habit.
Consider the following:

    class A(object):
        x = []
    a = A()
    a.x.append(3)

you would expect to change an object which is owned by a. But you change the class variable instead. Explicit is always better than implicit ;)

Would you welcome pull requests to get rid of this and similar?

**Comment by hut** [comment]
I did it for performance. Considering that this is only done for immutable values and the class variables are never changed, I'd prefer to leave it like this.

To give you some numbers: I ran ranger with --profile in both variants, 1. class-wise declaration and 2. explicit declaration in the `__init__` function. Then I typed `gujjjjjjjjjjj` to make ranger load my `/usr/*/`, creating 20623 File objects and 1773 directory objects in the process.

The time spent in the `__init__` functions of the directores increases from 0.180ms to 0.227ms and for the file objects it jumps from 0.262ms to 0.401ms, adding a total of 0.186ms loading time.

Note that this is on my i5 with an SSD where I don't care about performance. This becomes important on raspi's, older netbooks and low end servers.
